src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java

```diff
@@ -53,8 +53,8 @@ public class SecuritySettingsConfigurer {
         ".plugins-ml-model-group",
         ".plugins-ml-model",
         ".plugins-ml-task",
-        ".plugins-ml-conversation-meta",
-        ".plugins-ml-conversation-interactions",
+        ".plugins-ml-memory-meta",
```

Comment:
Keep these two old indices to make it safer?

Reply:
Yes that's doable. Actually let's keep them to avoid BWC problems.